### ISSUE TITLE
Missing Version in Help -> About of OpenShift Eclipse Che

### ISSUE THREAD

**Issue by apupier** [openshiftio/openshift.io#1160]
![image](https://user-images.githubusercontent.com/1105127/31774090-00b162b8-b4e5-11e7-96d0-b4d2b3c9a156.png)

**Comment by ibuziuk** [comment]
currently che version in the about dialog should be the same as in the status osio info:
![image](https://user-images.githubusercontent.com/1461122/31890017-0b07442e-b802-11e7-9598-1800d61aa6aa.png)
However, once che-server is multi-tenant it will not deployed [1]  to user space anymore and would be located in dedicated cluster. Need to get back to this issue once multi-tenant migration is done and decide how the version of che server would be obtained for osio / workspace UI

[1] http://central.maven.org/maven2/io/fabric8/tenant/apps/che/

**Comment by rhopp** [comment]
@apupier This is how the `About` dialog is looking now:
![image](https://user-images.githubusercontent.com/1215011/44910492-b364fa00-ad23-11e8-854d-8476aab0daa1.png)
As we are not doing any official releases, but continuous deployment, our maven artifacts are still version `1.0.0-SNAPSHOT`, but we also have the most important things there: Build Time and Revision (commit id).

I see, that having `-SNAPSHOT` version appearing in production is not nice. @ibuziuk, do you think we could do something about that? For example put the commit id (or maybe even better - date of build) there instead of `1.0.0-SNAPSHOT`?

Or could we put there version of Che, on which this rhche is based on (currently 6.10.0)?

**Comment by stevengutz** [comment]
Raising the priority so someone hopefully fixes/closes this old bug

**Comment by garagatyi** [comment]
Was fixes at some point, but no one tracked it to close down the issue. 
![image](https://user-images.githubusercontent.com/1851863/46919971-fe934d80-cfef-11e8-96d4-4022c65eeaaf.png)
![image](https://user-images.githubusercontent.com/1851863/46919974-03580180-cff0-11e8-9ac7-314ab013e88f.png)

Maybe we need a bot that would ask to close the issue and then ppl involved would react and validate whether the issue is still actual?

**Comment by ibuziuk** [comment]
I believe we should close this issue now, since all the data provided in the About dialog is correct.

> I see, that having -SNAPSHOT version appearing in production is not nice. @ibuziuk, do you think we could do something about that? For example put the commit id (or maybe even better - date of build) there instead of 1.0.0-SNAPSHOT?

I agree, but we should address and discuss in the separate issue IMO, since currently it is not clear what would be the best approach for displaying version of rh-che (we might also consider removing it completely since we are doing CD and version = revision in our case)

> Maybe we need a bot that would ask to close the issue and then ppl involved would react and validate whether the issue is still actual?

Not sure it will help much since this might just create extra noise in the issue, but not solve the root of the problem - that the issues should be triaged on a regular basis.
